php/public/toggle-dark-mode.js

```diff
@@ -0,0 +1,26 @@
+// Function to toggle theme
+function toggleTheme() {
+    const currentTheme = document.documentElement.getAttribute('data-theme');
+    const newTheme = (currentTheme === 'dark') ? 'light' : 'dark';
+    document.documentElement.setAttribute('data-theme', newTheme);
+    localStorage.setItem('theme', newTheme);
+
+    // Change the icon based on the current theme
+    const themeIcon = document.getElementById('theme-icon');
+    themeIcon.textContent = newTheme === 'dark' ? '☀️' : '🌙'; // Switch between moon and sun icons
```

Comment:
I'm not a fan of emoji, their rendering is random and depends on the system...

Reply:
> their rendering is random and depends on the system...

yes and no
Meanwhile the display of emojis is in general as good as uniform in every browser, but some browser extensions (such as Chromoji) can affect the display though. However, since this project already makes a lot of use of emojis and all emoji fonts are already loaded in the CSS and these are two very common emojis that are an integral part of utf8 and are already displayed in every shell, for example, and it doesn't matter at all whether they are displayed exactly in one way or another for this purpose, I wanted to leave it as it is.
I can of course also use two svg elements, but I found that to be absolutely unnecessary considering what I wanted to achieve and display.

Reply:
I am fine eith the emoji 👍